errors/threw-undefined.md

```diff
@@ -0,0 +1,10 @@
+# Threw undefined in render
+
+#### Why This Error Occurred
+
+Sometime during render you `threw undefined`. Since this isn't a valid error there isn't a stack trace. We show this error instead to let you know what to look for.
```

Comment:
```suggestion
Somewhere in your code you `throw` an `undefined` value. Since this isn't a valid error there isn't a stack trace. We show this error instead to let you know what to look for.
```